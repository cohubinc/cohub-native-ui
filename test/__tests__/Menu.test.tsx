import React from "react";
import { mount } from "enzyme";

import { Menu } from "dist";

it("renders without crashing", async () => {
  const wrapper = mount(
    <Menu
      appearance="light"
      items={[
        { label: "Sales Orders", action: null },
        { label: "Purchase Orders", action: null }
      ]}
    />
  );

  expect(wrapper).toMatchSnapshot();
});
