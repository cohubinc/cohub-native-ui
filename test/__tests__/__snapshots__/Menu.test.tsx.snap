// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<Menu
  appearance="light"
  items={
    Array [
      Object {
        "action": null,
        "label": "Sales Orders",
      },
      Object {
        "action": null,
        "label": "Purchase Orders",
      },
    ]
  }
>
  <div
    className="p-05 flex MenuContainer flex-row justify-evenly items-center"
    style={
      Object {
        "backgroundColor": "hsl(0, 0%, 98%)",
        "borderRadius": "8px",
      }
    }
  >
    <div
      className="px-05 py-05 pointer"
      key="Sales Orders"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "hsla(0, 0%, 0%, 0%)",
          "borderRadius": "4px",
        }
      }
    >
      <Typography
        color="hsl(0, 2%, 22%)"
      >
        <Component
          color="hsl(0, 2%, 22%)"
        >
          <span
            className=""
            data-qa="text"
            style={
              Object {
                "color": "hsl(0, 2%, 22%)",
                "fontFamily": "Inter",
                "fontSize": "14px",
                "fontWeight": 400,
                "textAlign": "initial",
                "textTransform": "initial",
              }
            }
          >
            Sales Orders
          </span>
        </Component>
      </Typography>
    </div>
    <div
      className="px-05 py-05 pointer"
      key="Purchase Orders"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "hsla(0, 0%, 0%, 0%)",
          "borderRadius": "4px",
        }
      }
    >
      <Typography
        color="hsl(0, 2%, 22%)"
      >
        <Component
          color="hsl(0, 2%, 22%)"
        >
          <span
            className=""
            data-qa="text"
            style={
              Object {
                "color": "hsl(0, 2%, 22%)",
                "fontFamily": "Inter",
                "fontSize": "14px",
                "fontWeight": 400,
                "textAlign": "initial",
                "textTransform": "initial",
              }
            }
          >
            Purchase Orders
          </span>
        </Component>
      </Typography>
    </div>
  </div>
</Menu>
`;
